include/boost/gil/concepts/color_base.hpp

```diff
@@ -120,24 +135,25 @@ struct ColorBaseConcept
         gil_function_requires<CopyConstructible<ColorBase>>();
         gil_function_requires<EqualityComparable<ColorBase>>();
 
-        typedef typename ColorBase::layout_t::color_space_t color_space_t;
+        using color_space_t = typename ColorBase::layout_t::color_space_t;
         gil_function_requires<ColorSpaceConcept<color_space_t>>();
 
-        typedef typename ColorBase::layout_t::channel_mapping_t channel_mapping_t;
+        using channel_mapping_t = typename ColorBase::layout_t::channel_mapping_t;
         // TODO: channel_mapping_t must be an MPL RandomAccessSequence
 
         static const int num_elements = size<ColorBase>::value;
 
-        typedef typename kth_element_type<ColorBase, num_elements - 1>::type TN;
-        typedef typename kth_element_const_reference_type<ColorBase, num_elements - 1>::type CR;
+        using TN = typename kth_element_type<ColorBase, num_elements - 1>::type;
+        using RN = typename kth_element_const_reference_type<ColorBase, num_elements - 1>::type;
```

Comment:
I was expecting to review reformatting changes only. Here you are changing a (public) member name from 'CR' to 'RN'. I assume you checked that this isn't used anywhere, right ?
But even if it isn't used, isn't it part of the public API, i.e. could some downstream code be affected ?

Reply:
I've opened a bug report about it #174